advise_reporter.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# thoth-storages
+# Copyright(C) 2020 Kevin Postlethwait, Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This is run periodically to provide metrics regarding Advise provided to Thoth Users."""
+
+import asyncio
+import logging
+import faust
+import os
+import ssl
+
+from thoth.lab import adviser
+
+from messages.advise_justification import AdviseJustificationMessage
+
+_LOGGER = logging.getLogger("thoth.advise_reporter")
+
+_KAFKA_BOOTSTRAP_SERVERS = os.getenv("KAFKA_BOOTSTRAP_SERVERS", "localhost:9092")
+_KAFKA_CAFILE = os.getenv("KAFKA_CAFILE", "ca.crt")
+KAFKA_CLIENT_ID = os.getenv("KAFKA_CLIENT_ID", "thoth-messaging")
+KAFKA_PROTOCOL = os.getenv("KAFKA_PROTOCOL", "SSL")
+KAFKA_TOPIC_RETENTION_TIME_SECONDS = 60 * 60 * 24 * 45
+ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=_KAFKA_CAFILE)
+app = faust.App("thoth-messaging", broker=_KAFKA_BOOTSTRAP_SERVERS, ssl_context=ssl_context, web_enabled=False)
```

Comment:
This looks like shared configuration for all the components that will use thoth-messaging. Can this be logic placed into one place - `thoth-messaging` module itself?

Reply:
The same way as the logging is setup for all the components that log. The setup should be lazy not to increase time of imports.

Reply:
https://github.com/thoth-station/messaging/pull/100/files#diff-f262f1b1016b774b837759e34a9781d0R29-R38

Reply:
Yes, as soon as that PR is merged and new thoth-messaging is released, I will use that functionality and remove the redundant part here.